icicle/tests/test_hash_api.cpp

```diff
@@ -878,18 +912,260 @@ TEST_F(HashApiTest, MerkleTreeLarge)
   }
 }
 
-#ifdef POSEIDON
+#ifdef POSEIDON2
 // p = 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001
 
   #include "icicle/fields/field_config.h"
-  #include "icicle/hash/poseidon_constants/bn254_poseidon.h"
 
 using namespace field_config;
-using namespace poseidon_constants_bn254;
+
+  #include "icicle/hash/poseidon2.h"
+
+// // DEBUG. This test could run only with bn254 curve. Dont remove!!!
```

Comment:
so why not put under ifdef BN254?

Reply:
Leon wants tests to run for all fields/curves
This test is there only if debug is needed in the future
The functionality of this test is covered by other tests